source/options.css

```diff
@@ -31,13 +22,13 @@ h3,
 h4 {
 	width: 100%;
 	margin-top: 0;
-	margin-bottom: 0.25em;
+	margin-bottom: 0.25rem;
 }
 
 hr {
 	margin: 1rem -1rem;
 	border: none;
-	border-bottom: solid #d3d3d3 1px;
```

Comment:
https://github.com/fregante/webext-base-css/blob/5e49453e04d8b97eee3fed983877a773b0df7b76/webext-base.css#L43

This whole rule can be dropped. _Maybe_ you only need vertical margins (we don't have them in RGH)

Reply:
That would be a good addition to `webext-base-css`, WDYT?

Reply:
What? Margins? Horizontal margins are already in base-css.

Reply:
Reasonable enough, but we do need vertical margins in this case.